Clarify documentation and workshop sections about updating and querying status (#35)
syntasso/kratix-docs: docs/main/05-reference/05-resources/04-status.md

requester. Kratix will automatically inject the required fields for status into
the `api`, you do not have to manually add these fields.

> Your `configure` pipeline can retrieve the existing status of a Resource by
> querying the resource as an input to the container, this helps to ensure
> that updating the status is idempotent within your [workflows](workflows).
> 

Reviewer: ```suggestion
Your `configure` pipeline can retrieve the existing status of a Resource by
querying the resource provided in input dir `/kratix/input/object.yaml`, this helps to ensure
that updating the status is idempotent within your [workflows](workflows).
```

is it also worth adding an example (e.g. pipeline creates s3 bucket, sets status, next run we read the status and just copy it across)? I feel the workshop does a good job of covering it, but not this page as a standalone doc
